docs/source/usage.rst

```diff
@@ -76,44 +81,55 @@ For information on installing ``repo2docker``, see :ref:`install`.
    Docker must be running on your machine in order to build images
    with ``repo2docker``.
 
-Building an image
-~~~~~~~~~~~~~~~~~
+The simplest invocation of ``repo2docker`` performs two steps:
+
+1. builds a Docker image from a git repo
+2. runs a Jupyter server within the image
 
-The simplest invocation of ``repo2docker`` builds a Docker image
-from a git repo, then runs a Jupyter server within the image
-so you can explore the repository's contents.
-You can do this with the following command::
+This two step process enable you to build an image and run it so you can
+explore the repository's contents.
+
+The **command** used is::
+
+  juptyer-repo2docker <URL to repo>
+
+where ``<URL to repo>`` provides a URL to the source repository.
+
+For example, use the following to build an image and launch a Jupyter Notebook
+server::
 
   jupyter-repo2docker https://github.com/jakevdp/PythonDataScienceHandbook
 
-After building (it might take a while!), it should output a message in your terminal::
+When the example completes building (which may take a few minutes), a message will
+be output to your terminal::
 
   Copy/paste this URL into your browser when you connect for the first time,
   to login with a token:
       http://0.0.0.0:36511/?token=f94f8fabb92e22f5bfab116c382b4707fc2cade56ad1ace0
 
 If you copy/paste that URL into your browser you will see a Jupyter Notebook with the
-contents of the repository you have just built!
+contents of the source repository which you have just built.
 
-Debugging the build process
----------------------------
+Using the ``--debug`` and ``--no-build`` parameters
+---------------------------------------------------
 
 If you want to debug and understand the details of the docker image being built,
-you can pass the ``debug`` parameter to the commandline. This will print the
-generated ``Dockerfile`` before building and running it.
+you can pass the ``debug`` parameter to the commandline:
 
   .. code-block:: bash
 
-  jupyter-repo2docker --debug https://github.com/jakevdp/PythonDataScienceHandbook
+     jupyter-repo2docker --debug https://github.com/jakevdp/PythonDataScienceHandbook
 
+This will print the generated ``Dockerfile``, build it, and run it.
 
-If you only want to see the ``Dockerfile`` output but not actually build it,
-you can also pass ``--no-build`` to the commandline. This ``Dockerfile`` output
-is for **debugging purposes** only - it can not be used by docker directly.
+To see the generated ``Dockerfile`` without actually building it,
+pass ``--no-build`` to the commandline. This ``Dockerfile`` output
+is for **debugging purposes** for ``repo2docker`` only - it can not
```

Comment:
**debugging purposes** for ``repo2docker`` only -> **debugging purposes** of ``repo2docker`` only ?